src/main/resources/bridge-sdk.properties

```diff
@@ -1,5 +1,5 @@
 log.level = warn
-sdk.version = 1
+sdk.version = 2
```

Comment:
Are we making breaking API changes at this point? Or is this just a matter of snapping the Java SDK at the same time as the server API?